.github/workflows/ci.yml

```diff
@@ -0,0 +1,206 @@
+name: ACS Community Packaging CI
+
+on:
+  pull_request:
+    branches:
+      - master
+      - release/**
+      - feature/**
+      - fix/**
```

Comment:
pay attention here that the branch filter for `pull_request` has not the same meaning of the `push` event - here you usually need only master or release if I understood correctly the branching model of this repo

Reply:
The reasoning to replicate all the filters is that I think it could make sense to keep the `feature/**` and `fix/**` PRs builds in the - rare but existing - cases where a complex feature is being developed over multiple feature branches to have smaller, more contained reviews and then merged into a final feature branch where all the changes are integrated one by one, before eventually being merged into the master or appropriate release branch.

Given that I've seen this kind of things happen in the past, if it does no harm I'd be glad to keep all the filters. Let me know your thoughts.